net/uacme/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2019 Lucian Cristian <lucian.cristian@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uacme
+PKG_VERSION:=1.0.15
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ndilieto/uacme/tar.gz/upstream/$(PKG_VERSION)?
+PKG_HASH:=b22e001928dd80249d7245b0cb61281ed487f8f1bd337b1264a754acf347561d
+
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-upstream-$(PKG_VERSION)
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+PKG_CONFIG_DEPENDS:= \
+  CONFIG_LIBCURL_GNUTLS \
+  CONFIG_LIBCURL_MBEDTLS \
+  CONFIG_LIBCURL_OPENSSL \
+  CONFIG_LIBCURL_WOLFSSL
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uacme
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+libcurl +LIBCURL_WOLFSSL:libmbedtls
+  TITLE:=lightweight client for ACMEv2
+  URL:=https://github.com/ndilieto/uacme
+endef
+
```

Comment:
I like this, also this can be used for LIBCURL_NOSSL because if I add @!LIBCURL_NOSSL then we will encounter the error mentioned above about circular dependency